Mathlib/LinearAlgebra/FreeModule/PID.lean

```diff
@@ -422,6 +422,10 @@ theorem Module.free_of_finite_type_torsion_free' [Module.Finite R M] [NoZeroSMul
   exact Module.Free.of_basis b
 #align module.free_of_finite_type_torsion_free' Module.free_of_finite_type_torsion_free'
 
+theorem Module.free_iff_noZeroSMulDivisors [Module.Finite R M] :
```

Comment:
Idem dito?

Reply:
Maybe. I'll try later (or maybe in #9345)